backend/app/infrastructure/contracts/erc20.py

```diff
@@ -22,6 +22,10 @@ def transfer(self, sender, receiver: str, amount: int):
         return self.w3.eth.send_raw_transaction(signed_tx.rawTransaction)
 
     def approve(self, owner, benefactor, wad: int):
+        print("owner of lock: ", owner)
+        print("owner address: ", owner.address)
```

Comment:
@paulperegud I did not remove these as it has a potential to help in case of debugging failed runs. If you think otherwise please let me know